.env.stg

```diff
@@ -33,8 +33,10 @@ REACT_APP_GTM_ID=
 REACT_APP_GOOGLE_RECAPTCHA_KEY=6Lc7yg0iAAAAALhS4C4Ez5hqMBub8hKdFvvr9sfc
 REACT_APP_POOL_FARM_BASE_URL=https://pool-farm.kyberswap.com
 
-REACT_APP_LIMIT_ORDER_API_WRITE=https://limit-order.stg.kyberengineering.io/write/api
-REACT_APP_LIMIT_ORDER_API_READ=https://limit-order.stg.kyberengineering.io/read-ks/api
+# REACT_APP_LIMIT_ORDER_API_WRITE=https://limit-order.stg.kyberengineering.io/write/api
+# REACT_APP_LIMIT_ORDER_API_READ=https://limit-order.stg.kyberengineering.io/read-ks/api
```

Comment:
why not delete

Reply:
will revert when finished test production api on stg